docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+| name                | 1.2.3.4                                                                                                                                                                                              |
+| port_details        | None                                                                                                                                                                                                 |
+| port_id             | None                                                                                                                                                                                                 |
+| project_id          | ef20bcc4215d49ddbf33e5c5740d89fe                                                                                                                                                                     |
+| qos_policy_id       | None                                                                                                                                                                                                 |
+| revision_number     | 0                                                                                                                                                                                                    |
+| router_id           | None                                                                                                                                                                                                 |
+| status              | DOWN                                                                                                                                                                                                 |
+| subnet_id           | None                                                                                                                                                                                                 |
+| tags                | []                                                                                                                                                                                                   |
+| updated_at          | 2025-02-04T13:13:38Z                                                                                                                                                                                 |
++---------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+In particular, we note down the value returned for the field `floating_ip_address`, which in this case corresponds to the dummy value `1.2.3.4`.
```

Comment:
You are using the `1.2.3.4` many times, in these cases I would usually use `$FLOATING_IP` as a placeholder, IDK if it s better method.


I also feel that this tutorial is pretty advanced. I think this section might be possible to  replaces with `log in with the ubuntu user to floating ip of the instances.

Reply:
I commented something like that before. Instead of using dummy values, refer to environment variables in the examples.
It is also important to explicitly tell the user to declare the variables, what to put there, and an example of command. Otherwise people run the commands with empty variables.